modules/node-shared/src/main/scala/io/constellationnetwork/node/shared/modules/SharedServices.scala

```diff
@@ -84,12 +87,17 @@ object SharedServices {
 
       currencyEventsCutter = CurrencyEventsCutter.make[F](None)
 
+      currencySnapshotValidationErrorStorage <- ValidationErrorStorage.make[F, CurrencySnapshotEvent, Option[BlockRejectionReason]](
+        cfg.validationErrorStorage.size
```

Comment:
does it compile? you've never provided this configuration variable 🤔

Reply:
I does, the configuration variable does not affect the compilation process.